app/models/feed.rb

```diff
@@ -3,6 +3,12 @@
 class Feed < BaseModel
   DEFAULT_FILE_NAME = 'feed-rss.xml'.freeze
 
+  AUDIO_MIME_TYPES = {
+    'mp3' => 'audio/mpeg',
+    'flac' => 'audio/flac',
+    'wav' => 'audio/wav'
+  }.freeze
```

Comment:
This was quickly added for the audio formats we support, as it doesn't need to be a general solution for all mime types